src/main/kotlin/solve/scene/view/FrameView.kt

```diff
@@ -23,12 +21,13 @@ class FrameView(
     private val scale: DoubleProperty,
     private val coroutineScope: CoroutineScope,
     private val associationsManager: AssociationsManager,
+    buffer: IntArray,
     frame: VisualizationFrame?
 ) : Group() {
     private var landmarksViews: Map<Layer, List<LandmarkView>>? = null
     private var drawnImage: Image? = null
     private var currentFrame: VisualizationFrame? = null
-    private val imageCanvas = Canvas(width, height)
+    private val canvas = BufferedCanvas(width, height, scale.value, buffer)
```

Comment:
looks like change from other request, isn't it?

Reply:
> looks like change from other request, isn't it?

These are changes that occured after rebase from main